packages/create-vite/index.js

```diff
@@ -3,7 +3,7 @@
 // @ts-check
 const fs = require('fs')
 const path = require('path')
-const argv = require('minimist')(process.argv.slice(2))
+const argv = require('minimist')(process.argv.slice(2), { string: ['_'] })
```

Comment:
I found it's hard to understand what's going on. We'd better add a comment for that, or just use `toString()` on usage.

Reply:
@antfu
Thanks for your comment!

I agree with it's hard to undarstand what's going on. So, I've added a comment for that.
Could you review that?

Thanks.